### PR TITLE
Add SwiftUI extensions to Lasso

### PR DESCRIPTION
**Pull request by cpisciotta**
Related PR: Updates #35 
Issue: #32 

Add extensions to `onAppear`, `onDisappear`, and `onTapGesture`, `onLongTapGesture`, and to `Alert.Button`.

### CODE REVIEW

File: Sources/Lasso/SwiftUI+Lasso/SwiftUI+Bindings.swift [hunk 1]
```diff
@@ -53,6 +54,25 @@ extension Button {
         )
     }
     
+    public init<Target: ActionDispatchable>(_ target: Target, animatedAction: Target.Action, label: () -> Label) {
+        self.init(action: {
+            withAnimation {
+                target.dispatchAction(animatedAction)
+            }
+        }, label: label)
```
Comment (g-mark) [inline Sources/Lasso/SwiftUI+Lasso/SwiftUI+Bindings.swift:62]: Nit: maybe use the same formatting as the one below?

**Review by g-mark — APPROVED** [review]
Looks great, @cpisciotta .  just one minor formatting nit.

**Comment by cpisciotta** [comment]
@g-mark Should be ready to go. I don't have write permissions, so you'll have to merge it in whenever you'd like.

**Comment by g-mark** [comment]
Thanks, @cpisciotta !

**Pull request merged**